MTM-48465 - documenation for tenant sso configuration access (#635)
SoftwareAG/c8y-docs: content/users-guide/administration-bundle/changing-settings.md

To enable the feature, the administrator must configure a connection with the authorization server. This is done in the Administration application.

##### Configuration access

SSO configurations can be configured to be exclusively accessible by the {{< management-tenant >}}, thus preventing other tenants from accessing the configurations.
Users of such tenants are unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from logging in via SSO.

> The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants. For more information about configuration access, please refer to the [Login options API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/putAccessLoginOptionResource) in the {{< openapi >}}.

Reviewer: ```suggestion
The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants. For more information about configuration access, refer to the [Login options API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/putAccessLoginOptionResource) in the {{< openapi >}}.
```

To enable the feature, the administrator must configure a connection with the authorization server. This is done in the Administration application.

##### Configuration access

SSO configurations can be configured to be exclusively accessible by the {{< management-tenant >}}, thus preventing other tenants from accessing the configurations.

> Users of such tenants will be unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from login via SSO.

Reviewer: ```suggestion
Users of such tenants are unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from logging in via SSO.
```

The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants, by assigning the following option to the tenants:

- category: `sso`
- key: `auth.sso.config.access.only.for.management`

With this option enabled (set to `true`), when a tenant tries to access an SSO configuration which is only accessible by the {{< management-tenant >}}, the access for that particular tenant is denied.

##### Configuration view

> Click the **Single sign-on** tab (this tab is only shown for tenants which have access to the SSO configuration) in the **Authentication** page.

Reviewer: ```suggestion
Click the **Single sign-on** tab in the **Authentication** page.
Note that the tab is only visible for tenants which have access to the SSO configuration. 
```
